lib.sh

```diff
@@ -0,0 +1,26 @@
+# scale up worker nodes to at least the provided number of nodes
+# Inputs:
+#  1: number of minimum worker nodes
+function scale_worker_nodes() {
+  echo "Reconcile workers to at least ${1} nodes"
```

Comment:
Maybe it would be good to put this line here. It would force the user to pass the argument
```
num_workers=${1:?Pass num_workers as arg[1]}
```

Aside that, was there any agreement on having all stuff in this repo in Python? If not, I'd just merge this and use it as is.

Reply:
Done.

I don't think there was an agreement on using python, in this case, we would be required to use Kube clients and manage dependencies, etc, not ideal for this case.